### PR TITLE
Adding the subscribe and unsubscribe options for the Google Group

### PR DESCRIPTION
**Pull request by jjahearn22**
Jake's draft at adding the subscribe and unsubscribe options for the Google Group to this webpage

### CODE REVIEW

File: content/contact/_index.md [hunk 1]
```diff
@@ -13,3 +13,9 @@ toc:
 We want to hear from you. All development efforts have been performed openly and we are seeking your feedback on our progress to date. Will these machine-readable formats and guidance aid your organization in going through the authorization process efficiently? Do you have any further ideas to enhance the work? Let us know!
 
 If you have questions or feedback, please provide comments either via email to [oscal@fedramp.gov](mailto:oscal@fedramp.gov) or as a comment to an existing issue  within the [FedRAMP Automation repository](https://github.com/GSA/fedramp-automation/issues).
+
```
Comment (aj-stein-gsa) [inline content/contact/_index.md:16]: I would recommend a header like Mailing List so we can deep link to this section if people want to skip the general info above _or_ more importantly we can deeplink to these details.

```suggestion

# Mailing List

```

File: content/contact/_index.md [hunk 2]
```diff
@@ -13,3 +13,9 @@ toc:
...
+If you are interested in joining the Google Group for the FedRAMP OSCAL Implementers meetings (held biweekly on Wednesdays at noon ET), please email [oscal-implementer+subscribe@fedramp.gov](mailto:oscal-implementer+subscribe@fedramp.gov) and you will be added to the group and on the calendar invitation. The Google Group provides an opportunity for the PMO to reach this community without mass emails or waiting for meetings, a repository with documented responses from the FedRAMP Automation team to group members, and a forum where individuals can support each other and ask questions.
+
+Should you find that you are no longer interested in being a part of the Google Group, please email [oscal-implementer+unsubscribe@fedramp.gov](mailto:oscal-implementer+unsubscribe@fedramp.gov) and you will be removed.
```
Comment (aj-stein-gsa) [inline content/contact/_index.md:19]: I would prefer if we keep the details solution-agnostic if possible and just refer to the mailing list, if we ever switch away from Google Group in GSA and use other equivalent options.

```suggestion
If you are interested in joining the mailing list for the FedRAMP OSCAL Implementers meetings (held biweekly on Wednesdays at noon ET), please email [oscal-implementer+subscribe@fedramp.gov](mailto:oscal-implementer+subscribe@fedramp.gov) and you will be added to the list and on the calendar invitation. The mailing list provides an opportunity for FedRAMP to reach this community without mass emails or waiting for meetings, a repository with documented responses from the FedRAMP Automation Team to group members, and a forum where individuals can support each other and ask questions.

Should you find that you are no longer interested in being a part of the mailing list, please email [oscal-implementer+unsubscribe@fedramp.gov](mailto:oscal-implementer+unsubscribe@fedramp.gov) and you will be removed.
```

**Review by aj-stein-gsa — CHANGES_REQUESTED** [review]
Nice work. All my proposed changes are minor, but I think we can wrap this tomorrow morning if you take the suggestions or provide similar edits and push those changes up yourself.

File: content/contact/_index.md [hunk 3]
```diff
@@ -13,3 +13,9 @@ toc:
 We want to hear from you. All development efforts have been performed openly and we are seeking your feedback on our progress to date. Will these machine-readable formats and guidance aid your organization in going through the authorization process efficiently? Do you have any further ideas to enhance the work? Let us know!
```
Comment (DimitriZhurkin) [inline content/contact/_index.md:13]: I think there should be a comma in front of `and we are seeking your feedback`:
```
All development efforts have been performed openly, and we are seeking your feedback on our progress to date.
```

File: content/contact/_index.md [hunk 4]
```diff
@@ -13,3 +13,9 @@ toc:
...
 
 If you have questions or feedback, please provide comments either via email to [oscal@fedramp.gov](mailto:oscal@fedramp.gov) or as a comment to an existing issue  within the [FedRAMP Automation repository](https://github.com/GSA/fedramp-automation/issues).
+
+If you are interested in joining the Google Group for the FedRAMP OSCAL Implementers meetings (held biweekly on Wednesdays at noon ET), please email [oscal-implementer+subscribe@fedramp.gov](mailto:oscal-implementer+subscribe@fedramp.gov) and you will be added to the group and on the calendar invitation. The Google Group provides an opportunity for the PMO to reach this community without mass emails or waiting for meetings, a repository with documented responses from the FedRAMP Automation team to group members, and a forum where individuals can support each other and ask questions.
```
Comment (DimitriZhurkin) [inline content/contact/_index.md:17]: I'd suggest adding a comma in front of `and you will be added`:
```
please email [oscal-implementer+subscribe@fedramp.gov](mailto:oscal-implementer+subscribe@fedramp.gov), and you will be added
```

File: content/contact/_index.md (hunk 2 above)
Comment (DimitriZhurkin) [inline content/contact/_index.md:19]: Again, I'd suggest adding a comma in front of `and you will be removed`.

File: content/contact/_index.md [hunk 5]
```diff
@@ -13,3 +13,9 @@ toc:
...
+
+Should you find that you are no longer interested in being a part of the Google Group, please email [oscal-implementer+unsubscribe@fedramp.gov](mailto:oscal-implementer+unsubscribe@fedramp.gov) and you will be removed.
+
+If you have further questions please see the email and GitHub repository links above.
```
Comment (DimitriZhurkin) [inline content/contact/_index.md:21]: I'd suggest adding a comma before `please`:
```
If you have further questions, please see the email and GitHub repository links above.
```

**Review by DimitriZhurkin — CHANGES_REQUESTED** [review]

File: content/contact/_index.md (hunk 3 above)
Comment (aj-stein-gsa) [reply content/contact/_index.md:13]: @DimitriZhurkin, in the future, can you please use [the suggestion feature for such feedback](https://docs.github.com/en/pull-requests/collaborating-with-pull-requests/reviewing-changes-in-pull-requests/commenting-on-a-pull-request#adding-comments-to-a-pull-request)?

File: content/contact/_index.md (hunk 5 above)
Comment (aj-stein-gsa) [reply content/contact/_index.md:23]: @DimitriZhurkin, in the future, can you please use [the suggestion feature for such feedback](https://docs.github.com/en/pull-requests/collaborating-with-pull-requests/reviewing-changes-in-pull-requests/commenting-on-a-pull-request#adding-comments-to-a-pull-request)?

**Review by aj-stein-gsa — APPROVED** [review]

**Review by DimitriZhurkin — APPROVED** [review]

**Pull request merged**